src/lints/trait_removed_supertrait.ron

```diff
@@ -55,6 +55,6 @@ SemverQuery(
         "public": "public",
         "zero": 0,
     },
-    error_message: "A supertrait was removed from a trait.",
+    error_message: "A supertrait was removed from a trait, users of the trait can no longer assume it can also be used like its supertrait.",
```

Comment:
```suggestion
    error_message: "A supertrait was removed from a trait. Users of the trait can no longer assume it can also be used like its supertrait.",
```
this seemed to read a bit better to me